### DISCUSSION TITLE
Which is better tool to integrate with LLM in Python

### DISCUSSION THREAD

**Discussion by aniketkalamkar** [llmware-ai/llmware#83] [Polls]
I am exploring RAG based app using LLM and looking for some suggestion to head starr

**Comment by JessBerl** [comment]
Hi @aniketkalamkar,

We’ve built llmware to be incredibly easy to use so it’s easy to get started.  We hope that the llmware community agrees that it’s a superior python package for getting started with RAG based applications using LLMs and to scale into production.   I suggest checking out our RAG examples, if you have not already, to jump start your efforts.   If you have any specific questions on using llmware, please post them as Q&A and we’re happy to help!
